src/views/Dashboard.vue

```diff
@@ -216,4 +220,8 @@ export default {
 			margin-bottom: 2vh;
 		}
 	}
+	.button-start-conversation {
```

Comment:
Linting warning:

223:2  ✖  Expected empty line before rule  rule-empty-line-before


```suggestion
	}

	.button-start-conversation {
```